[2024-05-29T17:02:46.464+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-05-29T17:02:46.710+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: test_1.task_2 scheduled__2024-03-07T00:00:00+00:00 [queued]>
[2024-05-29T17:02:46.821+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: test_1.task_2 scheduled__2024-03-07T00:00:00+00:00 [queued]>
[2024-05-29T17:02:46.831+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 1
[2024-05-29T17:02:46.935+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): task_2> on 2024-03-07 00:00:00+00:00
[2024-05-29T17:02:47.017+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'test_1', 'task_2', 'scheduled__2024-03-07T00:00:00+00:00', '--job-id', '43', '--raw', '--subdir', 'DAGS_FOLDER/test.py', '--cfg-path', '/tmp/tmpku_mgq5p']
[2024-05-29T17:02:47.062+0000] {standard_task_runner.py:91} INFO - Job 43: Subtask task_2
[2024-05-29T17:02:47.005+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=283) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-05-29T17:02:47.112+0000] {standard_task_runner.py:63} INFO - Started process 319 to run task
[2024-05-29T17:02:47.637+0000] {task_command.py:426} INFO - Running <TaskInstance: test_1.task_2 scheduled__2024-03-07T00:00:00+00:00 [running]> on host 2832635d2b45
[2024-05-29T17:02:49.398+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='test_1' AIRFLOW_CTX_TASK_ID='task_2' AIRFLOW_CTX_EXECUTION_DATE='2024-03-07T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-03-07T00:00:00+00:00'
[2024-05-29T17:02:49.411+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-05-29T17:02:49.448+0000] {logging_mixin.py:188} INFO - Run function_2!
[2024-05-29T17:02:49.454+0000] {python.py:237} INFO - Done. Returned value was: None
[2024-05-29T17:02:49.463+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-05-29T17:02:49.608+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=test_1, task_id=task_2, run_id=scheduled__2024-03-07T00:00:00+00:00, execution_date=20240307T000000, start_date=20240529T170246, end_date=20240529T170249
[2024-05-29T17:02:49.798+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-05-29T17:02:49.861+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
